runtime/js/99_main.js

```diff
@@ -935,6 +936,7 @@ function bootstrapMainRuntime(runtimeOptions, warmup = false) {
       core.wrapConsole(globalThis.console, core.v8Console);
     }
 
+    event.setEventTargetData(globalThis);
```

Comment:
Can you explain why does this needs to happen during `bootstrap` calls and not during snapshotting?